package.json

```diff
@@ -42,8 +44,7 @@
     "button"
   ],
   "dependencies": {
-    "ember-cli-babel": "^5.1.6",
-    "ember-cli-htmlbars": "1.0.3"
```

Comment:
not sure why I moved it that time as the 2.4.* outputs don't show it there https://github.com/ember-cli/ember-addon-output/blob/v2.4.3/package.json#L26